### ISSUE TITLE
License for Open Source?

### ISSUE THREAD

**Issue by JustinGrote** [ironmansoftware/powershell-pro-tools#43]
Hi @adamdriscoll ,

First off, thank you so much for you and your team's tireless efforts to the PowerShell community. Your commercial solutions are the single greatest value-add I can possibly recommend to PowerShell users, and I do it often.

Due to the deprecation announcement, will the license be MIT when the code is published as per this existing repo or something else? I would like to request a noncommercial redistributable license (MIT or similar) as I would like to preserve some of your great work and port it to the vscode extension where it makes sense, of course attributing and giving you full credit.

Let me know! Thanks, and best of success to your other fantastic products.

**Comment by adamdriscoll** [comment]
Thanks, @JustinGrote! I appreciate the comment. 

The license will continue to be MIT. I'd like to donate this body of work back to the community it whatever form it may take. I think many of the VS Code features could be moved right into the PowerShell extension. Once it's released, you'll see some of the workarounds we had to put in place to make certain things work that would be much easier if it was directly integrated.

**Comment by adamdriscoll** [comment]
Initial commit is done. The builds are going to fail until I can solve some issues: https://github.com/ironmansoftware/powershell-pro-tools/issues/45

I removed PSPackager and PSScriptPad. PSScriptPad has been replaced with Windows Form Designer which is arguably the most popular feature of PSScriptPad. PSScriptPad and PSPackager both used Actipro components and their licensing won't work in an open source project like this.
